### PR TITLE
Create warning

### PR DESCRIPTION
**Pull request by guillermosanta**

### CODE REVIEW

File: api/migrations/20210513022753-create-user.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 "use strict";
 module.exports = {
   up: async (queryInterface, Sequelize) => {
-    await queryInterface.createTable("Users", {
+    await queryInterface.createTable("users", {
```
Comment (chungthuang) [inline api/migrations/20210513022753-create-user.js:4]: Thank you for making it lower case!

File: src/plugins/UtilCommands/controllers/commands/getWarnings.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+exports.conf = {
+  aliases: ["addcustom"],
+  cooldown: 5,
+  permissions: ["ADMINISTRATOR"],
+  arguments: [],
+};
+
+exports.run = async (client, message, args) => {
+  const warnings = await Warning.findAll({ where: { userId: message.author.id } });
+  console.log(warnings);
```
Comment (chungthuang) [inline src/plugins/UtilCommands/controllers/commands/getWarnings.js:19]: What is the goal of this function? Is it to log something in the server?

File: src/plugins/UtilCommands/controllers/commands/getWarnings.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+const { User, Warning } = require("./../../../../../api/models/index.js");
```
Comment (chungthuang) [inline src/plugins/UtilCommands/controllers/commands/getWarnings.js:1]: Usually a long relative path(`./../../../../..`) is not recommended.

File: src/plugins/UtilCommands/controllers/commands/getWarnings.js (hunk 2 above)
Comment (guillermosanta) [reply src/plugins/UtilCommands/controllers/commands/getWarnings.js:19]: I made this command to test how I would be able to retrieve the warnings a user had. It's in the UtilCommands plugin, and the command will not be there once this is pushed to production.

File: src/plugins/UtilCommands/controllers/commands/getWarnings.js (hunk 3 above)
Comment (guillermosanta) [reply src/plugins/UtilCommands/controllers/commands/getWarnings.js:1]: Do you recommend using an absolute path?

Comment (chungthuang) [reply src/plugins/UtilCommands/controllers/commands/getWarnings.js:1]: I'm not familiar with the JS import system. Can you use absolute path?

**Pull request merged**